druid/src/widget/flex.rs

```diff
@@ -700,20 +700,6 @@ impl Spacing {
     }
 }
 
-// we have these impls mostly for our 'flex' example, but I could imagine
-// them being broadly useful?
```

Comment:
I didn't know what to do with this comment

Reply:
you can just delete that.